packages/twenty-front/src/modules/activities/emails/right-drawer/components/RightDrawerEmailThread.tsx

```diff
@@ -83,6 +96,24 @@ export const RightDrawerEmailThread = () => {
   const lastMessage = visibleMessages[visibleMessagesCount - 1];
   const subject = visibleMessages[0]?.subject;
 
+  const canReply = useMemo(() => {
+    return currentUser && lastMessage && messageThreadExternalId != null;
+  }, [currentUser, lastMessage, messageThreadExternalId]);
+
+  const handleReplyClick = () => {
+    if (!canReply) {
+      return;
+    }
+
+    const userEmail = currentUser?.email;
+    const url = `https://mail.google.com/mail/?authuser=${userEmail}#all/${messageThreadExternalId}`;
```

Comment:
Here it would help to put not the userEmail but the connect account related to this thread.

For example I connected a second gmail account and when I clicked on reply it tried to open the first default demo tim@apple.dev account (which doesn't exist)

Reply:
In my case, if I connect a second gmail account with then default tim@apple.dev, emails dont get imported for the second gmail account. Idk if thats a bug from my end.
I am running these cron jobs - 
`npx nx run twenty-server:command cron:messaging:messages-import`
`npx nx run twenty-server:command cron:messaging:message-list-fetch`
These jobs are it right? or are there anymore?
But it does make sense to have connected account related to the thread to be populated here.
Issue is I couldnt find a connected account related to that thread or message table.
How do i get the connected account related to that thread?
Need help :(

Reply:
You have to run the worker with : 

`yarn nx worker twenty-server` (it will process the jobs)

Then create the cron with : `npx nx run twenty-server:command cron:messaging:messages-import`

Reply:
I do run worker using `npx nx run twenty-server:worker` and then cron jobs.
Ill check them once more however I cant find a connected account related to that thread or message table.
How do i get the connected account related to that thread?